sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/cloudfoundry/environment/Constants.java

```diff
@@ -6,47 +6,52 @@
 /**
  * utils class of Constants
  */
-public class Constants {
+final class Constants {
+
+    private Constants() {
+
+    }
+
     // Property namespaces
     /**
      * ServiceBus namespace
      */
-    public static final String NAMESPACE_SERVICE_BUS = "azure.servicebus";
+    static final String NAMESPACE_SERVICE_BUS = "azure.servicebus";
 
     /**
      * Document DB namespace
      */
-    public static final String NAMESPACE_DOCUMENTDB = "azure.documentdb";
+    static final String NAMESPACE_COSMOSDB = "azure.documentdb";
 
     // VCAP credential key names
     /**
      * Shared access key name
      */
-    public static final String SHARED_ACCESS_NAME = "shared_access_key_name";
+    static final String SHARED_ACCESS_NAME = "shared_access_key_name";
```

Comment:
do we need to change
from `SHARED_ACCESS_NAME` to `SHARED_ACCESS_KEY_NAME`

Reply:
we can do that